lib/seattleflu/id3c/cli/command/etl/fhir.py

```diff
@@ -158,7 +208,7 @@ def create_encounter_resource(encounter_identifier: List[dict],
         "resourceType": "Encounter",
         "class": encounter_class,
         "identifier": encounter_identifier,
-        "status": "finished",
+        "status": encounter_status,
         "period": {
             "start": encounter_date,
             "end": encounter_date,
```

Comment:
Do we need to also change the period if the `Encounter.status` is different?

Reply:
Ah, right. If a status is not finished, we would not want an end date for the period.

Reply:
Oh but wait. I remember why I didn't adjust encounter period based on the hospitalization start or end date. 
We're not tracking the hospitalization period. That's a separate entity. We're only interested in the encounter where a swab was taken. So that always has a start/end date of the swab collection date.

Reply:
This contradicts the encounter status change I made, though. Perhaps we should continue setting all Encounter.status to 'finished'.

Reply:
Huh, then shouldn't the status of the encounter always be finished (if this is tracking the encounter where the swab was taken)?

Reply:
Exactly :smile:

Reply:
Hmm. Jover just informed me on Slack she found this in the FHIR docs:
> The duration recorded in the period of this encounter covers the entire scope of this hospitalization record.

https://www.hl7.org/fhir/encounter-definitions.html#Encounter.hospitalization

If we stay true to the docs and change our Encounter.period.start and Encounter.period.end to reflect the hospitalization start/end times, we will need to update the FHIR ETL to extract a date other than start date (if available) for our swab collection (ID3C encounter) date.

Reply:
The other option is to keep the Encounter pointed at when/where the swab was taken and move the hospitalization details to QuestionnaireResponse

Reply:
That's also a possibility.

If we lean toward more structure (using Encounter.hospitalization), it looks like we should be able to use [Specimen.collection](https://www.hl7.org/fhir/specimen-definitions.html#Specimen.collection) to store collection date.

Reply:
Even though it's slightly more work, I would prefer to use Encounter.hospitalization, Encounter.status and Encounter.period (accordingly), and Specimen.collection. But I also understand if we want to punt on this for now.

Reply:
Yeah, I feel like we should follow FHIR, but we might want to punt it depending on how much ROR work we're getting...

Reply:
I'm ok with marking the Encounter.period as a future `FIXME`.

Good find on the Encounter.partOf. I agree, it sounds like unnecessary work, and I _really_ don't want to process a nested Encounter resource right now. I'd prefer to kick this can down the road.